learn/guides/distinctive-language-features/data.md

```diff
@@ -980,7 +978,7 @@ To access every element in the children of ``e``, for every element ``e`` in ``x
 
 To access every text item in the children of ``e``, for every element ``e`` in ``x``, use **`x/*.text()`**.
 
-To access every element named ``para`` in the descendants of ``e``, for every element ``e`` in ``x``, use **``x/**/<para>``**. Here the use of ``**`` signifies any number of levels within an xml element.
+To access every element named ``para`` in the descendants of ``e``, for every element ``e`` in ``x``, use **``x/**/<para>``**. Here the use of ``**`` signifies any number of levels within a XML element.
```

Comment:
Given that this is an abbreviation and the first letter is `X`, this should actually use "an" right?

```suggestion
To access every element named ``para`` in the descendants of ``e``, for every element ``e`` in ``x``, use **``x/**/<para>``**. Here the use of ``**`` signifies any number of levels within an XML element.
```